auth4genai/snippets/get-started/cloudflare-agents-js/call-your-api.mdx

```diff
@@ -42,6 +42,27 @@ npm install hono \
 
 In the root directory of your project, copy the `.dev.vars.example` into `.dev.vars` file and configure the Auth0 and OpenAI variables.
 
+```bash .dev.vars wrap lines
+# ...
+# You can use any provider of your choice supported by Vercel AI
+OPENAI_API_KEY="OPENAI API KEY"
+
+SESSION_STORE=cloudflare-kv
+SESSION_STORE_NAMESPACE=Session
```

Comment:
these two variables are not defined in the starter's readme: https://github.com/priley86/cloudflare-agents-starter/blob/dc00c33799b26683c501580e47a14d26353c16dd/README.md?plain=1#L61-L75

Are they needed?